code/modules/atmospherics/gasmixtures/gas_types.dm

```diff
@@ -212,7 +212,7 @@
 
 /datum/gas/hydrogen
 	id = "hydrogen"
-	specific_heat = 15
+	specific_heat = 300
```

Comment:
Uh this might be a bad idea

Reply:
Why that?

Reply:
Hydrogen and Tritium fires will be much weaker. The bombs made from these gases will be ruined. The production of tritium will be harder because it will have lower temperatures